helix-view/src/editor.rs

```diff
@@ -165,6 +165,8 @@ pub struct Config {
     pub indent_guides: IndentGuidesConfig,
     /// Whether to color modes with different colors. Defaults to `false`.
     pub color_modes: bool,
+    /// Whether to render rainbow highlights. Defaults to `false`.
+    pub rainbow_brackets: bool,
```

Comment:
It defaults to true not false.

Reply:
quoting the PR description:

> Note: while this PR is in draft state, rainbow brackets are temporarily enabled by default.

Reply:
Defaulting to true is temporary - before opening un-marking this PR as a draft I will drop the commit that defaults this to true

Reply:
I wonder if it would be better to default this to true? I wonder in what case this is not a good idea? It aids readability and doesn't really have much downside beside being bad for poeple with color blindness.

Reply:
I think it's a somewhat controversial feature - some people really like it and some would really hate it on by default. For features like that, I think the convention is to have it disabled by default.